src/models/util/ParseCalChartUtils.ts

```diff
@@ -0,0 +1,93 @@
+export function ReadFourCharCode(buffer: DataView, offset: number): string {
+  return String.fromCharCode(
+    buffer.getUint8(offset + 0),
+    buffer.getUint8(offset + 1),
+    buffer.getUint8(offset + 2),
+    buffer.getUint8(offset + 3)
+  );
+}
+
+export function ReadTwoCharCode(buffer: DataView, offset: number): string {
+  return String.fromCharCode(
+    buffer.getUint8(offset + 0),
+    buffer.getUint8(offset + 1)
+  );
+}
+
+export function ReadInt32(buffer: DataView, offset: number): number {
+  return buffer.getInt32(offset);
+}
+
+export function ReadInt16(buffer: DataView, offset: number): number {
+  return buffer.getInt16(offset);
+}
+
+export function ReadStringTillEnd(buffer: DataView, offset: number): string {
+  let count = 0;
+  let found = '';
```

Comment:
For variables that will eventually be returned, and you can't think of anything better, I recommend using `retVal` as the name.
```suggestion
  let retVal = '';
```

Reply:
Done!